tests/pgautofailover_utils.py

```diff
@@ -1070,6 +1070,22 @@ def get_synchronous_standby_names(self):
 
         return out.strip()
 
+    def get_synchronous_standby_names_local(self):
+        """
+            Gets number sync standbys  via pg_autoctl
```

Comment:
Just noticed that this, as well as the `get_synchronous_standby_names` docstring, are copies of the `get_number_sync_standbys` docstring.